pkgs/applications/networking/instant-messengers/hangups/default.nix

```diff
@@ -0,0 +1,88 @@
+{ pkgs, stdenv, python3 }:
+let 
+  packageOverrides = self: super:{
+    ConfigArgParse = python3.pkgs.buildPythonPackage rec {
```

Comment:
Please use `super.ConfigArgParse.override` and do that for the other ones as well